imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -568,9 +568,9 @@ func Test_fdFdstatSetFlags(t *testing.T) {
 	// Restore the O_APPEND flag.
 	requireErrnoResult(t, wasip1.ErrnoSuccess, mod, wasip1.FdFdstatSetFlagsName, uint64(fd), uint64(wasip1.FD_APPEND))
 	require.Equal(t, `
-==> wasi_snapshot_preview1.fd_fdstat_set_flags(fd=4,flags=1)
+==> wasi_snapshot_preview1.fd_fdstat_set_flags(fd=4,flags=APPEND)
 <== errno=ESUCCESS
-`, "\n"+log.String())
+`, "\n"+log.String()) // FIXME? flags==1 prints 'flags=APPEND'
```

Comment:
according to docs, flags is an fdflags type, so usually in log formatting we format the flag name vs have people need to look up what it was and do the bit flag math in their head. Maybe it was surprising that the logging "fixed" in a different change? https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md#-fdflags-flagsu16